src/CardInfo.js

```diff
@@ -38,66 +42,57 @@ class CardInfo extends React.Component {
-                            <div class="card-info-select">
-                                <Select
-                                    defaultValue={{value: status, label: status}}
-                                    options={statusOptions}
-                                    onChange={this.handleChangeStatus}
-                                />
-                            </div>
-                            <TextEditor
-                                text={title}
-                                onChangeText={this.handleChangeTitle}
-                            />
-                            <TextEditor
-                                text={description}
-                                onChangeText={this.handleChangeDescription}
-                            />
-                            <ListChildrenCard childrenCards={getCardsByArrayId(children, cards)} />
-                            <SelectorChilds 
-                                cards={cards} 
-                                idParent={id} 
-                                addChilds={this.handleAddChilds}
-                                createNewCard={this.handleCreateNewCard}
+                {isChanging
+                    ? <Fragment>
+                        {parent && <Link 
+                            parentId={parent._id} 
+                            onClick={this.handleOnClickLink}
+                        >
+                            {parent.title}
+                        </Link>}
+                        <div class="card-info-heap-avatar">
+                            <HeapAvatars size={46} id={_id} />
+                        </div>
+                        <div class="card-info-select">
+                            <Select
+                                value={{value: status, label: status}}
+                                options={statusOptions}
+                                onChange={this.handleChangeStatus}
                             />
-                        </Fragment>
-                        : <div>
-                            <p>{`Status: ${status}`}</p>
-                            <p>{`Title: ${title}`}</p>
-                            <p>{description}</p>
                         </div>
-                    }
-                </div>
+                        <TextEditor
+                            text={title}
+                            onChangeText={this.handleChangeTitle}
+                        />
+                        <TextEditor
+                            text={description}
+                            onChangeText={this.handleChangeDescription}
+                        />
+                        <ListChildrenCard 
+                            childrenCards={getCardsByArrayId(children, cards)}
+                            onRedirect={onRedirect}
+                        />
+                        <SelectorChilds 
+                            cards={cards} 
+                            idParent={_id} 
```

Comment:
why it's `idParent` not `parentId`?

Reply:
I don't correct it. Its were added last PR